### PR TITLE
test: Add tests missing in PR #1143

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* #972

### CODE REVIEW

File: source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.Application/UseCases/Factories/CompletedBatchFactory.cs [hunk 1]
```diff
@@ -14,18 +14,19 @@
...
 using Energinet.DataHub.Wholesale.Batches.Interfaces.Models;
 using Energinet.DataHub.Wholesale.IntegrationEventPublishing.Application.BatchAggregate;
+using Energinet.DataHub.Wholesale.IntegrationEventPublishing.Application.UseCases.Mappers;
 using NodaTime.Extensions;
 
-namespace Energinet.DataHub.Wholesale.IntegrationEventPublishing.Application.UseCases.Mappers;
+namespace Energinet.DataHub.Wholesale.IntegrationEventPublishing.Application.UseCases.Factories;
 
-public class CompletedBatchMapper : ICompletedBatchMapper
+public class CompletedBatchFactory : ICompletedBatchFactory
```
Comment (lasrinnil) [inline source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.Application/UseCases/Factories/CompletedBatchFactory.cs:22]:
```suggestion
public static class CompletedBatchFactory : ICompletedBatchFactory
```

Nit

File: source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.Infrastructure/Persistence/IntegrationEventPublishingDatabaseContext.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ public IntegrationEventPublishingDatabaseContext()
     {
     }
 
-    public DbSet<CompletedBatch> CompletedBatches { get; private set; } = null!;
+    public virtual DbSet<CompletedBatch> CompletedBatches { get; private set; } = null!;
```
Comment (lasrinnil) [inline source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.Infrastructure/Persistence/IntegrationEventPublishingDatabaseContext.cs:37]: Why `virtual`? Please educate me! :D

File: source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.UnitTests/Application/UseCases/PublishCalculationResultsHandlerTests.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+using Test.Core;
+using Xunit;
+
+namespace Energinet.DataHub.Wholesale.IntegrationEventPublishing.UnitTests.Application.UseCases;
+
+public class PublishCalculationResultsHandlerTests
+{
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task Foo(
```
Comment (lasrinnil) [inline source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.UnitTests/Application/UseCases/PublishCalculationResultsHandlerTests.cs:33]: Please keep this Test name <3

**Review by lasrinnil — DISMISSED** [review]

File: source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.Infrastructure/Persistence/IntegrationEventPublishingDatabaseContext.cs (hunk 2 above)
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.Infrastructure/Persistence/IntegrationEventPublishingDatabaseContext.cs:37]: Required in order to be able to override in test.

File: source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.UnitTests/Application/UseCases/PublishCalculationResultsHandlerTests.cs (hunk 3 above)
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/IntegrationEventPublishing/IntegrationEventPublishing.UnitTests/Application/UseCases/PublishCalculationResultsHandlerTests.cs]: Thx. Fixed.

**Review by lasrinnil — DISMISSED** [review]

**Review by lasrinnil — DISMISSED** [review]

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**